File: OPP/Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
  Map<String, Map<String, Integer>> typeCarAccepted;
  ArrayList<String> seatsMaterial;

  public UberVan(String license, Account driver) {
    super(license, driver);
  }

  @Override
  public void setPassanger(Integer passenger) {
    if (passenger == 6) {
      this.passenger = passenger;
    } else {
      System.out.println("Necesitas 6 pasajeros");
    }
  }

}

File: OOP-js-0/PHP/uberVan.php
<?php
require_once('car.php');
class UberVan extends Car {
  public $typeCarAccepted;
  public $seatsMaterial;
  protected $passengers;

  public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial) {
    parent::__construct($license, $driver);
    $this->typeCarAccepted = $typeCarAccepted;
    $this->seatsMaterial = $seatsMaterial;
}

  public function getPassengers() {
    return $this->passengers;
  }


  public function setPassengers($passengers) {
    if($passengers == 6) {
      $this->passengers = $passengers;
    } else {
      echo "Necesitas 6 pasajeros";
      echo "<br>";
    }
  }
}

?>